server/remote_cache/content_addressable_storage_server/content_addressable_storage_server.go

```diff
@@ -657,6 +660,10 @@ func (s *ContentAddressableStorageServer) GetTree(req *repb.GetTreeRequest, stre
 				}
 			} else {
 				log.Warningf("Not caching incomplete tree cache")
+				if _, migration := cache.(*migration_cache.MigrationCache); migration {
```

Comment:
I don't think this is useful. We already see this warning for the existing cache so this is going to give you false positives.

Reply:
I added this metric so we can make sure that the number of not found errors goes down over time, which we'd expect as more data gets copied. Was seeing a lot of this error when testing and assumed it was from data not being copied yet, so wanted to make sure there wasn't a permanent problem if this stays high. Seemed fine if the count includes false positives as long as it trends down - do you disagree?